### DISCUSSION TITLE
Latest release 1.5.14 still not in CWS nor AMO

### DISCUSSION THREAD

**Discussion by darkred** [openstyles/stylus#1121] [General]
It's been almost a week but unfortunately the latest release of Stylus, 1.5.14, is still not in [CWS](https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne) nor [AMO](https://addons.mozilla.org/en-US/firefox/addon/styl-us/) .
Maybe there has been some issue with the submissions? 
If I am not mistaken, the CWS review process has been improved and is no longer that slow, and AMO review was never too slow .
Regarding CWS, maybe it's something like [this](https://github.com/sindresorhus/refined-github/issues/3739) of Refined GitHub extension ?

**Comment by narcolepticinsomniac** [comment]
Lots of code changes, so only a CWS beta release for now. We've gotten some useful feedback, and it'll probably be a little while before we submit updates to stable in either extension gallery.

**Comment by darkred** [comment]
Thank you for the quick response.

**Comment by arogl** [comment]
For the ambitious could you create Firefox and chrome packages much like is done for [uBlock](https://github.com/gorhill/uBlock/blob/master/.github/workflows/main.yml) and [ViolentMonkey](https://github.com/violentmonkey/violentmonkey/blob/master/.github/workflows/release.yml)

**Comment by tophf** [comment]
Unlike those extensions Stylus can be used directly from the source. It doesn't have to be built. The ambitious can download it from github anytime without our help.
